indico/modules/attachments/api/util.py

```diff
@@ -27,9 +27,17 @@
 from MaKaC.common.contextManager import ContextManager
 
 
+def get_event(linked_object):
+    from MaKaC.conference import Category, Conference
+
+    if isinstance(linked_object, Conference):
```

Comment:
How do you end up with Conference objects here? I don't think that should happen..